main_routers/system_router.py

```
@@ -0,0 +1,555 @@
+            
+            logger.info(f"Steam后台已配置 {num_achievements} 个成就: {achievements}")
+            return JSONResponse(content={"count": num_achievements, "achievements": achievements})
+        except Exception as e:
+            logger.error(f"获取成就列表失败: {e}")
+            return JSONResponse(content={"error": str(e)}, status_code=500)
+    else:
+        return JSONResponse(content={"error": "Steamworks未初始化"}, status_code=500)
+
+
+@router.get('/api/file-exists')
+async def check_file_exists(path: str = None):
```

Ruff (0.14.8)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)